bitwarden_license/src/Commercial.Core/Billing/ProviderBillingService.cs

```diff
@@ -476,4 +476,38 @@ public async Task<ProviderSubscriptionDTO> GetSubscriptionDTO(Guid providerId)
 
         await providerPlanRepository.ReplaceAsync(providerPlan);
     };
+
+    public async Task<ProviderPaymentInfoDTO> GetPaymentInformationAsync(Guid providerId)
+    {
+        var provider = await providerRepository.GetByIdAsync(providerId);
+
+        if (provider == null)
+        {
+            logger.LogError(
+                "Could not find provider ({ID}) when retrieving payment information.",
+                providerId);
+
+            return null;
+        }
+
+        if (provider.Type == ProviderType.Reseller)
+        {
+            logger.LogError("payment information cannot be retrieved for reseller-type provider ({ID})", providerId);
+
+            throw ContactSupport("Consolidated billing does not support reseller-type providers");
+        }
+
+        var taxInformation = await paymentService.GetTaxInfoAsync(provider);
+        var billingInformation = await paymentService.GetBillingAsync(provider);
```

Comment:
`GetBillingAsync` is the method we discussed in our Dev Sync that we determined does a little too much work for what's asked of it. We'd also like to try and phase out the `StripePaymentService` as much as possible. What do you think about adding two new methods to the `SubscriberService`?

`SubscriberService.GetTaxInformation` and `SubscriberService.GetPaymentMethod`.  You could add new DTOs representing tax information and a payment method in our Core/Billing folder so we'd have ownership of them. Then, the `ProviderPaymentInfoDTO` could use those two new DTOs.

Reply:
Sorry might have missed something, but I think I got pinged for a re-review. Did we have an update here?